generators/entity/index.js

```diff
@@ -667,7 +667,6 @@ class EntityGenerator extends BaseBlueprintGenerator {
                 context.entityModelFileName = context.entityFolderName;
                 context.entityParentPathAddition = this.getEntityParentPathAddition(context.clientRootFolder);
                 context.entityPluralFileName = entityNamePluralizedAndSpinalCased + context.entityAngularJSSuffix;
-                context.entityServiceFileName = context.entityFileName;
```

Comment:
I think you didn't read this part of comment 'However, we should also not remove the entityServiceFileName variable and it's current assignment to be equal to the entityFileName as it might have been used in a module/blueprint. We can do the clean in the next major release.'